crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -244,8 +242,15 @@ public ITimeProvider TimeProvider
         /// </summary>
         public float ViewerHeightAboveWater { get; private set; }
 
+        List<LodDataMgr> _lodDatas = new List<LodDataMgr>();
+
+        List<OceanChunkRenderer> _oceanChunkRenderers = new List<OceanChunkRenderer>();
+
         SampleHeightHelper _sampleHeightHelper = new SampleHeightHelper();
 
+        public List<WaterBody> WaterBodies => _waterBodies;
+        List<WaterBody> _waterBodies = new List<WaterBody>();
```

Comment:
It might be easier to have this as a static list on the WaterBody component instead. This is how Unity does it with the NavMeshComponents.

Reply:
yeah that made things better in a few ways - good shout!